test/controllers/tasks/active.psc.details.controller.unit.ts

```diff
@@ -1,14 +1,17 @@
+import { urlUtils } from "../../../src/utils/url";
```

Comment:
move the urlUtils import so it is with the other imports, but make sure that the import mocks is still the first import, safer to add to bottom of imports.

Reply:
Ah my bad, it did that automatically